frontend/studio-root/pages/Contact/Contact.tsx

```diff
@@ -55,6 +56,20 @@ export const Contact = () => {
               </Paragraph>
             </div>
           </section>
+          <section className={classes.section}>
+            <GitHub />
```

Comment:
The GitHub icon doesn't look like the one in the sketches. It should have a dark blue background like the other icons:

![a](https://github.com/Altinn/altinn-studio/assets/24462611/0b45d0c0-f174-466e-bae4-78fca77c58f0)

![b](https://github.com/Altinn/altinn-studio/assets/24462611/eaea4771-306f-4224-bcbf-601bb9c5b42d)

Reply:
Here is an example of how you can fix it: https://github.com/Altinn/altinn-studio/commit/f41082af72d4a728db33979bc23046b202d2d53e